bin/export.js

```diff
@@ -5,7 +5,7 @@ const fs = require('fs');
 const schema = require('../schema');
 const exportedSchema = schema.exportSchema();
 
-fs.writeFile(
+fs.writeFileSync(
```

Comment:
This is to remove the `Calling an asynchronous function without callback is deprecated` warning.